AvaliadorDisciplinas.FRONTEND/src/pages/Disciplina/index.jsx

```diff
@@ -38,35 +48,41 @@ function DisciplinaIndex() {
         </div>
         <div className={styles.filter}>
           <input
+            className={styles.input}
             type="text"
-            id="nome"
             placeholder={
               "Procure por " +
               atributosDisciplina()
-                .find((item) => item.value === atributo)
-                .text.toLowerCase()
+                .find((item) => item.value === attribute.value)
+                .label.toLowerCase()
             }
             onChange={(e) => {
               setText(e.target.value);
             }}
           />
           <div className={styles.dropdown}>
-            <Dropdown
-              data={atributosDisciplina()}
+            <Select
+              className={styles.select}
```

Comment:
Acho que faz mais sentido o Select bloquear a pesquisa no dropdown, já que só existem 2 escolhas.
```suggestion
              className={styles.select}
              isSearchable={false}
```

Reply:
Corrigido!